File: Auto-generated-tasks/Difficulty-1/Task-7/Solution.f90
integer:: Cnt = 6, q, Rez = 0
integer(len=Cnt):: T
do q = 0, Cnt
    if (T(q) < 199) then
        Rez = Rez + 1
    end if
end do
print *, "Rez=", Rez

File: Auto-generated-tasks/Difficulty-1/Task-7/Solution.cs
int Cnt = 6, q, Rez = 0;
int[] T = new int[Cnt];
for (q = 0; q < Cnt; q++) {
	if (T[q] < 199)
		Rez = Rez + 1;
}
Console.WriteLine("Rez=" + Rez);